src/frontend/src/pages/GanttPage/GanttPageFilter.tsx

```diff
@@ -0,0 +1,188 @@
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import UnfoldMoreIcon from '@mui/icons-material/UnfoldMore';
+import UnfoldLessIcon from '@mui/icons-material/UnfoldLess';
+import RestartAltIcon from '@mui/icons-material/RestartAlt';
+import { ChangeEvent, FC } from 'react';
+
+interface GanttPageFilterProps {
+  car1Handler: (event: ChangeEvent<HTMLInputElement>) => void;
+  car2Handler: (event: ChangeEvent<HTMLInputElement>) => void;
+  status: string;
+  statusHandler: (event: SelectChangeEvent) => void;
+  selectedTeam: string;
+  teamList: string[];
+  teamHandler: (event: SelectChangeEvent) => void;
+  currentStart: Date | null;
+  startHandler: (value: Date | null) => void;
+  currentEnd: Date | null;
+  endHandler: (value: Date | null) => void;
+  expandedHandler: (expanded: boolean) => void;
+  resetHandler: () => void;
+}
+
+const GanttPageFilter: FC<GanttPageFilterProps> = ({
+  car1Handler,
+  car2Handler,
+  status,
+  statusHandler,
+  teamHandler,
+  startHandler,
+  endHandler,
+  expandedHandler,
+  teamList,
+  selectedTeam,
+  currentStart,
+  currentEnd,
+  resetHandler
+}) => {
+  const carFilters = (
+    <Grid item container direction="column" xs={12} md={1} sx={{ justifyContent: 'start', alignItems: 'start' }}>
+      <Grid item xs={12} md={1}>
+        Car
+      </Grid>
+      <Grid item container xs={12} md={1}>
+        <Grid item>
+          <Checkbox
+            defaultChecked
+            icon={<LooksOneOutlinedIcon />}
+            checkedIcon={<LooksOneOutlinedIcon />}
+            onChange={car1Handler}
+            sx={{
+              color: 'white',
+              '&.Mui-checked': {
+                color: '#ef4345'
+              },
+              '& .MuiSvgIcon-root': {
+                fontSize: '2rem'
+              },
+              paddingLeft: 0
+            }}
+          />
+        </Grid>
+        <Grid item>
+          <Checkbox
+            defaultChecked
+            icon={<LooksTwoOutlinedIcon />}
+            checkedIcon={<LooksTwoOutlinedIcon />}
+            onChange={car2Handler}
+            sx={{
+              color: 'white',
+              '&.Mui-checked': {
+                color: '#ef4345'
+              },
+              '& .MuiSvgIcon-root': {
+                fontSize: '2rem'
+              },
+              paddingLeft: 0
+            }}
+          />
+        </Grid>
+      </Grid>
+    </Grid>
+  );
+
+  const buttons = (
+    <Grid
+      item
+      container
+      xs={12}
+      md={3}
+      spacing={1}
+      sx={{ justifyContent: 'end', alignItems: 'center', alignSelf: 'center', justifySelf: 'end' }}
+    >
+      <Grid item>
+        <Button
+          onClick={() => {
+            expandedHandler(true);
+          }}
+          startIcon={<UnfoldMoreIcon />}
+        >
+          Expand
+        </Button>
+      </Grid>
+      <Grid item>
+        <Button
+          onClick={() => {
+            expandedHandler(false);
```

Comment:
small bug here but it looks like when expanded=false the collapse (and reset?) button doesnt work and when expanded=true the expand button doesnt work. So what that means is if you manually expand or close an individual project gantt these buttons don't undo it. (Try with the default gantt opening a project and clicking collapse/reset) idk if there's a way to force a refresh or something